recipes/libxnvme/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+meson setup builddir --prefix=$PREFIX -Ddefault_library=shared -Dwith-spdk=false 
+meson compile -C builddir
+meson install -C builddir
+
+# Create activation and deactivation scripts
+mkdir -p $PREFIX/etc/conda/activate.d
+mkdir -p $PREFIX/etc/conda/deactivate.d
+
+# Activation script
```

Comment:
These scripts need to be removed, we won't merge this as long as it doesn't use the normal path like $PREFIX/bin

Reply:
ack. In case anybody else stumbles over this, then these scripts, that I thought were needed, are not, since the ``$MESON_ARGS`` as @xhochy suggested sets, among other things  ``-Dlib=...`` and ``--prefix`` which is what is needed for executables, and the pkg-config xnvme.pc to be located correctly.